documentation/source/documentation/cheatsheets/python.rst

```diff
@@ -614,3 +614,163 @@ Python collects any number of keyword arguments into a dictionary.  In
+|                                                  |        loop(count - 1)                            |
+|                                                  |      end                                          |
+|                                                  |    end                                            |
++--------------------------------------------------+---------------------------------------------------+
+
+break
+^^^^^
+
++--------------------------------------------------+---------------------------------------------------+
+| Python                                           | Dylan                                             |
++==================================================+===================================================+
+| .. code-block:: python                           | .. code-block:: dylan                             |
+|                                                  |                                                   |
+|    for i in range(10):                           |    block(return)                                  |
+|      if i == 5:                                  |      for (i from 0 below 10)                      |
+|        break                                     |        if (i = 5) return() end;                   |
+|      print(i)                                    |        format-out("%d\n", i);                     |
+|                                                  |      end;                                         |
+|                                                  |    end                                            |
++--------------------------------------------------+---------------------------------------------------+
+
+continue
+^^^^^^^^
+
++--------------------------------------------------+---------------------------------------------------+
+| Python                                           | Dylan                                             |
++==================================================+===================================================+
+| .. code-block:: python                           | .. code-block:: dylan                             |
+|                                                  |                                                   |
+|    for i in range(5):                            |    for (i from 0 below 5)                         |
+|      if i == 2:                                  |      block(return)                                |
+|        continue                                  |        if (i = 2) return() end;                   |
+|      print(i)                                    |        format-out("%d\n", i);                     |
+|                                                  |      end;                                         |
+|                                                  |    end                                            |
```

Comment:
Might as well name the exit function `continue`.  Also maybe name it `break` in the break example?